ehr/test/src/org/labkey/test/pages/ehr/NotificationAdminPage.java

```diff
@@ -104,6 +104,7 @@ public void clickManuallyTriggerEmail(String name)
     {
         log("Manually Triggering Email for " + name);
         name = name.replace(" ", "");
+        scrollIntoView(Locator.tagWithAttributeContaining("a", "id", name).withText("Manually Trigger Email"));
         clickAndWait(Locator.tagWithAttributeContaining("a", "id", name).withText("Manually Trigger Email"), 0);
```

Comment:
Avoid finding elements more than once. `scrollIntoView` returns the `WebElement` that was scrolled into view, which can be clicked without finding the element again.
```suggestion
        scrollIntoView(Locator.tagWithAttributeContaining("a", "id", name).withText("Manually Trigger Email"))
                .click();
```

Also, `clickAndWait` without a wait value is misleading.